vspace/src/main/webapp/WEB-INF/views/staff/exhibit/config.jsp

```diff
@@ -0,0 +1,38 @@
+<%@ page pageEncoding="UTF-8"%>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="t"%>
+<%@ taglib prefix="sec"
+	uri="http://www.springframework.org/security/tags"%>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form"%>
+
+<h1>Exhibition Configuration</h1>
+
+<div style="padding-bottom: 20px;">
+	<c:url value="/staff/exhibit/config" var="postUrl" />
+	<c:if test="${param.success eq true}">
+    	<div class="alert alert-success">
+  			<strong>Successfully saved!</strong>
+		</div>
+	</c:if>
+	<c:if test="${param.success eq false}">
+    	<div class="alert alert-danger">
+  			<strong>Error: Could not save default space!</strong>
+		</div>
+	</c:if>
+	Select your exhibition and default space here.
+	<form:form method="POST"
+		action="${postUrl}?${_csrf.parameterName}=${_csrf.token}">
+		<select class="form-control" name="exhibitionParam">
```

Comment:
this needs a label that says: "Select the exhibition you would like to configure:"